reana_client/cli/files.py

```diff
@@ -67,10 +73,12 @@ def get_files(ctx, workflow, _filter,
     Examples: \n
     \t $ reana-client ls --workflow myanalysis.42
     """
+    JSON = 'json'
+    URL = 'url'
+    FILE_PATH = "https://reana.cern.ch/api/workflows/%s/workspace/%s"
```

Comment:
Is this `reana.cern.ch` the only possible address? Or should we use the `current_rs_api_client.swagger_spec.api_url` value?

Reply:
To get the URL you can use [`ctx.obj`](https://github.com/diegodelemos/reana-client/blob/master/reana_client/cli/__init__.py#L87) which contains the configuration with the `REANA_SERVER_URL` set by the user: [`ctx.obj. reana_server_url`](https://github.com/diegodelemos/reana-client/blob/master/reana_client/cli/__init__.py#L29) :)